time timestamp
float64 measured_width
float64 width_error
float64 stitch_time
float64 process_time
bool deviation
float64 tcp_x
float64 tcp_y
float64 tcp_z